src/java.desktop/share/classes/javax/swing/text/html/HTMLEditorKit.java

```diff
@@ -1182,6 +1181,15 @@ public void handleEndOfLineString(String eol) {
      * A factory to build views for HTML.  The following
      * table describes what this factory will build by
      * default.
+     * <p>
```

Comment:
I'd rather move this block below the table. The sentence above introduces the table, then it lists how the common HTML tags are handled. This added paragraph describes how unsupported and unknown tags are handled.

Does it make sense to use lists?

* When the container is editable, …
* When the container is not editable,
  * If tag is recognized but not supported, …
  * If tag is unknown or not supported, …

Or simply split the two cases in different paragraphs for better readability?

Reply:
I added the list formatting for better readability.